legend-engine-xt-relationalStore-bigquery-execution-tests/src/test/java/org/finos/legend/engine/plan/execution/stores/relational/connection/test/ExternalIntegration_TestConnectionAcquisitionWithFlowProvider_BigQuery.java

```diff
@@ -86,32 +74,27 @@ public static void verifyTestSetup()
     }
 
     @BeforeClass
-    public static void setupTest() throws IOException
+    public static void setupTest()
     {
         Vault.INSTANCE.registerImplementation(new EnvironmentVaultImplementation());
     }
 
-    @Override
-    protected Subject getSubject()
-    {
-        return null;
-    }
-
     @Before
-    public void setup()
-    {
-        LegendDefaultDatabaseAuthenticationFlowProvider flowProvider = new LegendDefaultDatabaseAuthenticationFlowProvider();
-        LegendDefaultDatabaseAuthenticationFlowProviderConfiguration flowProviderConfiguration = LegendDefaultDatabaseAuthenticationFlowProviderConfiguration.Builder.newInstance()
-                .withAwsConfig(awsConfig)
-                .withGcpWorkloadConfig(gcpWorkloadConfig)
-                .build();
+    public void setup() throws IOException {
+        RelationalConnectionTest rtb = new RelationalConnectionTest();
```

Comment:
no need to created rtb objects, as this class extends RelationalConnectionTest, so its protected methods are directly available, like how testConnection is available